Makefile

```diff
@@ -56,6 +57,17 @@ test-with-cover:
 	$(GOTEST) $(GOTEST_OPT_WITH_COVERAGE) $(ALL_PKGS)
 	go tool cover -html=coverage.txt -o coverage.html
 
+.PHONY: addlicense
+addlicense:
+	@ADDLICENCESEOUT=`$(ADDLICENCESE) -y 2019 -c 'OpenTelemetry Authors' $(ALL_SRC) 2>&1`; \
```

Comment:
Not sure if this runs because currently (or maybe I am wrong) we don't have all the files with license.

Reply:
It only works for Go source files you'd try deleting the license from one of Go source files and It will be added automatically after "make addlicense"